workspaces/orchestrator/plugins/orchestrator-common/src/openapi/api-doc-template.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: orchestrator-backend
+  description: The Orchestrator backend plugin API
+  tags:
+    - orchestrator
+spec:
+  type: openapi
+  lifecycle: experimental
+  owner: development
+  definition: |
```

Comment:
what is the definition supposed to be?

Reply:
this is  a template file. the script generates the doc file from it. it appends the openapi spec